lib/install/dispatcher/dispatcher.go

```diff
@@ -99,12 +103,14 @@ type Status byte
 
 const (
 	// StatusUnknown indicates an unknown progress status
-	StatusUnknown Status = 0
+	StatusUnknown Status = iota
 	// StatusCompleted indicates a completed operation
-	StatusCompleted Status = iota
+	StatusCompleted
 	// StatusCompletedPending indicates a completed operation
 	// but with installer still active
-	StatusCompletedPending Status = iota
+	StatusCompletedPending
+	// StatusAborted indicates an aborted operation
+	StatusAborted
```

Comment:
This (and the related case) seems like new code?

Reply:
Yes, it is. I'll move this to another PR.